test/test_cases/suffix_comments_good.ex

```diff
@@ -0,0 +1,23 @@
+def test() do
+  # comment 1
+  if true do
+    # comment 2
+    if false do
+      nil
+      # comment 3
```

Comment:
Thanks for your suggestion, I agree that it looks better if the comment is before the `nil`. I'll see if I can reverse the ordering without any breaking changes.